builders/build/substrate/adding-custom-made-module.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Adding a Custom-Made Module
+description: Substrate is a modular blockchain framework that makes it easy to build unique and innovative Appchains composing built-in modules with custom-made ones.
+---
+
+# Adding a Custom-Made Module {: #adding-custom-made-module } 
+
+## Introduction {: #introduction }
+
+By providing a comprehensive library of pre-built modules addressing many common requirements, the framework greatly simplifies the process of building an Appchain and accelerates the deployment and evolution into a ContainerChain. However, addressing an innovative use case usually requires a development effort to fully meet the requirements, and, in Substrate, adding custom logic translates into writing and integrating runtime modules. 
+
+The example presented in the [Modularity](/learn/framework/modules/#custom-module-example){target=_blank} article shows a simple lottery module exposing two transactions:
+
+- **Buy tickets**
+- **Award prize**
+
+The implementation of those transactions also uses storage, emits events, defines custom errors, and relies on other modules to handle currency (to charge for the tickets and transfer the total amount to the winner) and randomize the winner selection.
+
+In this article, the following steps, necessary to build and add the example module to the runtime, will be covered: 
+
+1. Create the lottery module files (package)
+2. Configure the module's dependencies
+3. Adding custom logic
+4. Configure the runtime with the new module
+
+--8<-- 'text/common/not-for-production-code-guard.md'
+
+## Checking Prerequisites {: #checking-prerequisites }
+
+To follow the steps in this guide, you will need to have the following:
+
+- **Clone the Tanssi repository from Github**
```

Comment:
URL? Why you put it in bold?

Reply:
Oh, just assumed it was the default style for lists